### DISCUSSION TITLE
step 5

### DISCUSSION THREAD

**Discussion by cbhavya09** [skills/.github#247] [Test with Actions]
not able to move to step 5

**Comment by cbhavya09** [comment]
https://github.com/cbhavya09/ci-workflow   - can you check here what needs to be done.

**Reply by sinsukehlab** [reply]
Well, this "Test with Actions" course does have some issues.
> Once you turn on branch protection, Actions can no longer push directly to `main`. You'll need to open the next step on your own.

The next step file is `.github/steps/5-merge-your-pull-request.md`. Please open the file directly for now. The course maintainers will fix the issues in a while.

**Reply by meeowngit** [reply]
Hi @sinsukehlab, I am recently facing this issue right away :( 
> The next step file is .github/steps/5-merge-your-pull-request.md. Please open the file directly for now. The course maintainers will fix the issues in a while.

Now it is July month, and still, this issue remains.


## **Update**
Finally it has moved to Step5:
 Actually the problem was that I was creating new branches all time and merging them to main =) 
And It is actually wrong! When you start the skills course then you have to make all commits to this branch `start-markdown` not in `main`.  And then merge your pull-request to `main` (i.e Step 5)

Hopefully, someone finds this helpful.

**Reply by Daniel4Ordina** [reply]
Check if the generated code from copilot is actually passing validation when you do your push, if it fails then the exercise does not progress to the next step.

**Reply by akebu6** [reply]
I got the same issue, the last part of the skills challenge isn't being validated and keeps failing no matter what

**Reply by sinsukehlab** [reply]
@meeowngit @Daniel4Ordina It seems that your replies are sent to the wrong conversation.

**Reply by szey** [reply]
git pull --no-rebase origin main
  
↑Try this

**Reply by board817** [reply]
The workflow "4-copilot-comment.yml" greps the "comments.js" file for "Create web server" (case-sensitive). Per the following thread https://github.com/orgs/skills/discussions/306
